src/Panel/SqlLogPanel.php

```diff
@@ -64,7 +64,9 @@ public function initialize()
             $logger = new DebugLog($logger, $name, $includeSchemaReflection);
 
             $connection->logQueries(true);
-            $connection->setLogger($logger);
+            if (method_exists($connection, 'setLogger')) {
+                $connection->setLogger($logger);
+            }
```

Comment:
You are missing the "else" part which should call the old method `$connection->logger($logger);`